CodelabsOne/app/src/main/java/com/feylabs/codelab_one/Model/BookModel.kt

```diff
@@ -0,0 +1,53 @@
+package com.feylabs.codelab_one.Model
+
+import android.os.Parcel
+import android.os.Parcelable
+import com.feylabs.codelab_one.R
+
+
+class BookModel() : Parcelable {
```

Comment:
I dont use data class here because this class will contain and handle little bit complex operation that cant be managed by DataClass, 

i dont reallu know btw :)
but i getting issues on other files if i use dataclass here